src/Blockcore.Indexer.Angor/Sync/SyncTasks/ProjectsSyncRunner.cs

```diff
@@ -101,22 +101,17 @@ public override async Task<bool> OnExecute()
    {
       var script = Script.FromHex(output.ScriptHex);
 
-      var ops = script.ToOps();
+      var parsedData = new DataFromOps();
 
-      if (ops.Count != 3 ||
-          ops.First().Name != Op.GetOpName(OpcodeType.OP_RETURN) ||
-          ops[1].PushData.Length != 33 ||
-          ops[2].PushData.Length != 32)
+      if (!parsedData.TryParse(script.ToOps()))
          return null;
 
-      var founderKey = new PubKey(script.ToOps()[1].PushData);
-
       var checkForExistingProject = await AngorMongoDb.ProjectTable.AsQueryable()
-         .AnyAsync(_ => _.FounderKey == founderKey.ToHex());
+         .AnyAsync(_ => _.FounderKey == parsedData.FounderPubKey.ToHex());
```

Comment:
You don't wanna change the database? `FounderPubKey` well probably not because we will add it back later